birthdays-project/birthday/templates/users/index.html

```diff
@@ -0,0 +1,16 @@
+{% extends 'base.html' %}
+
+{% block content %}
+
+    <h3>Members with birthdays.</h3>
+    <ul>
+        {% for e in user %}
+            <li>
+                <a href="{% url 'user_detail' e.id %}">
+                    {# {{ e.id }} Used for testing#}
```

Comment:
General remark - don't check in commented code. If you need it for debugging, you _could_ put it inside an if/endif construct, like

```
{% if debug %}{{ e.id }} Used for testing{% endif %}
```

and you'd have to pass `debug` to the template, like (views.py)

```
from django.conf import settings

...

def index(request):
    userDB = User.objects.all()
    return render(request, 'users/index.html', {
        'user': userDB,
        'debug': settings.DEBUG,
    })
```